vhdbuilder/packer/generate-vhd-publishing-info.sh

```diff
@@ -14,7 +14,7 @@ for v in "${required_env_vars[@]}"
 do
     if [ -z "${!v}" ]; then
         if [ "$v" == "IMAGE_VERSION" ]; then
-           IMAGE_VERSION=$(date +%Y.%m.%d)
+           IMAGE_VERSION=$(date +%Y%m.%d.0)
```

Comment:
won't we also need to update how the automated image bump stuff determines the new SIG version as well? I need to review how that works exactly.

Reply:
should it be %Y.%m.%d.0? 
+%Y%m.%d.0 -> 202302.24.0, is that intended?

Reply:
yes it is.